packages/core/src/tooltip/Tooltip.css

```diff
@@ -7,10 +7,13 @@
 .saltTooltip {
   background: var(--tooltip-background);
   border-color: var(--saltTooltip-borderColor, var(--tooltip-status-borderColor));
-  border-width: var(--saltTooltip-borderWidth, var(--salt-size-border));
   border-style: var(--saltTooltip-borderStyle, var(--salt-container-borderStyle));
+  border-width: var(--saltTooltip-borderWidth, var(--salt-size-border));
   box-shadow: var(--saltTooltip-shadow, var(--salt-overlayable-shadow-popout));
   color: var(--saltTooltip-text-color, var(--salt-text-primary-foreground));
+  font-size: var(--saltTooltip-fontSize, var(--salt-text-label-fontSize));
```

Comment:
Why set it to label font size?

Reply:
Otherwise when used in `h3`, it will inherit bigger size? See reproduce link in description.

Reply:
But shouldn't it be `--salt-text-fontSize`? Why `label`?

Reply:
label is smaller than body text. `11px` vs `12px` on medium density

Reply:
Oh, my bad. Somehow i'm confused with the naming